addons/hr_holidays/tests/test_accrual_allocations.py

```diff
@@ -1389,6 +1389,8 @@ def test_yearly_cap(self):
                 'request_hour_to': '12',
             })
             self.assertEqual(leave.number_of_hours, 2)
+            allocation.invalidate_recordset(['leaves_taken'])
```

Comment:
Since, e-mails are not longer sent, less cache is invalidated and flushed. Here the field does not contain all dependencies to be invalidated. Similar to other changes below.
I'll add a comment in the commit message.